src/_imagingft.c

```diff
@@ -132,6 +132,27 @@ getfont(PyObject *self_, PyObject *args, PyObject *kw) {
         return NULL;
     }
 
+#if PY_MAJOR_VERSION > 3 || PY_MINOR_VERSION > 11
```

Comment:
Technically this is false for Python 4.0-4.11 and true again for Python 4.12+.

We'll drop support for Python 3.11 in 2027 and this conditional can go away. I don't expect a Python 4.0 before then but it's not a certainty.

Reply:
I think this is an OR, so it is correct for Python 4 but incorrectly includes Python 1.12 or Python 2.12, neither of which is likely to ever be released?

Reply:
I was thinking along @nulano's lines, that yeah, 1.12 and 2.12 aren't going to happen.

Reply:
All good then!

This is a good opportunity to bring up [PEP 404](https://peps.python.org/pep-0404/) (Python 2.8 not found) :)